Sunset DI log probes and prepare Live Debugger docs for GA (#37791)
DataDog/documentation: content/en/tracing/trace_collection/dynamic_instrumentation/expression-language.md

## Try your own conditions

This interactive simulator helps you experiment with the Expression Language syntax in a realistic environment. It shows how conditions affect whether data is captured when instrumenting a method.

> Select one of the examples or enter an expression in the "when" field and click "SIMULATE" to see if the log would be generated based on your condition.

Reviewer: Line 89 was generalized to "whether data is captured," but this line still says "the log would be generated." Align them:

```suggestion
Select one of the examples or enter an expression in the "when" field and click "SIMULATE" to see whether data is captured based on your condition.
```
